lib/puppet/provider/vcsrepo/git.rb

```diff
@@ -571,4 +578,8 @@ def git_with_identity(*args)
       git(*args)
     end
   end
+
+  def return_git_client_version
+    Facter.value('vcsrepo_git_ver')
```

Comment:
see @binford2k 's comment, it may be a better idea to use `git('--version')` which uses the `git()` function that's already available in the provider.